src/components/Checkout/PaymentWrapper.vue

```diff
@@ -47,6 +47,10 @@ export default {
 		amount: {
 			type: String,
 			default: ''
+		},
+		selectedOption: {
+			type: String,
+			default: ''
```

Comment:
please set the default to 'bt' we should ensure that is always provided and we just switch if the exp assignment differs.

Reply:
Also this is just to ensure this component can work without a prop being passed.